### PR TITLE
Create 2024-2-21-Add_a_Payment_Link

### PR DESCRIPTION
**Pull request by oliverquinlan**
Added documentation of payment link usability testing

# PR Checklist

- [ x] If you are proposing a new decision record document, used the right template for that
      - ([ADR](https://github.com/alphagov/forms/blob/main/ADR/ADRXXX-architecture-decision-record-template.md), [decision-record](https://github.com/alphagov/forms/blob/main/decision-record/DRXXX-decision-record-template.md), engagement, [research](https://github.com/alphagov/forms/blob/main/research/YYYY-MM-DD-template.md))
- [ x] Set yourself as the Assignee
- [ x] Tag anyone you would like to review, or @forms-design or @forms-devs
- [ x] Fill in the template below

---

# What

Added documentation of the 'Add a payment link' journey usability testing.

# How to review

Check the documentation represents the research.

Check for errors.

# Who can review

Forms team

### CODE REVIEW

File: research/2024-2-21-Add_a_Payment_Link [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Title
```
Comment (hannahkc) [inline research/2024-2-21-Add_a_Payment_Link:1]:
```suggestion
# Add a payment link - usability testing
```

File: research/2024-2-21-Add_a_Payment_Link [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Key Headlines 
+
+- The initial version of the feature was comprehensible to users and they appeared able to set up payment links in the Forms interface.
+- Users anticipate a time consuming process to get set up to use GOV.UK Pay involving others in their organisation such as Finance colleagues
```
Comment (hannahkc) [inline research/2024-2-21-Add_a_Payment_Link:6]:
```suggestion
- Users anticipate a time consuming process to get set up to use GOV.UK Pay involving others in their organisation such as finance colleagues.
```

File: research/2024-2-21-Add_a_Payment_Link [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- The long version of the content gave users the clearest understanding of the feature.
+- We identified a number of areas to iterate the content to improve the understanding of the feature.
+- Participants were happy to use the feature, with the caveat that they would need to get sign off and processes in place to use GOV.UK Pay.
+- There is a need for more guidance on how to set up GOV.UK Pay to accept a GOV.UK Forms generated reference number
+
+## Supporting Evidence
+- [Report](https://docs.google.com/presentation/d/1oTOkZVe8XtLRk7jOQczT0yg1uLXr0-phWsTHKh248pE/edit#slide=id.g2bb376953d2_0_0)
```
Comment (hannahkc) [inline research/2024-2-21-Add_a_Payment_Link:13]:
```suggestion
- [Report](https://docs.google.com/presentation/d/1oTOkZVe8XtLRk7jOQczT0yg1uLXr0-phWsTHKh248pE/edit?usp=sharing)
```

**Comment by oliverquinlan** [comment]
@hannahkc Thanks for spotting those issues, have committed your changes.

**Review by DavidBiddle — COMMENTED** [review]
Looks good to me (other than the file extension change Hannah mentioned!)

**Comment by oliverquinlan** [comment]
@hannahkc I can't commit your suggestions as I changed them and the filename manually and then pushed the change, outdating yours. Please could you re-review.

**Review by hannahkc — APPROVED** [review]

File: research/2024-2-21-Add_a_Payment_Link.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Add a payment link - usability testing
+
+## 2024-02-21 / Sprint 47
```
Comment (christophercameron-ixd) [inline research/2024-2-21-Add_a_Payment_Link.md:3]:
```suggestion
2024-02-21 / Sprint 47
```

File: research/2024-2-21-Add_a_Payment_Link.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Add a payment link - usability testing
```
Comment (christophercameron-ixd) [inline research/2024-2-21-Add_a_Payment_Link.md:1]:
```suggestion
# Usability testing: Add a payment link
```

**Review by christophercameron-ixd — DISMISSED** [review]
Looks good to me. Just 2 suggestions. Feel free to ignore

**Comment by oliverquinlan** [comment]
Thanks @christophercameron-ixd have committed those. Please could you OK the review if you are happy with it.

**Review by christophercameron-ixd — APPROVED** [review]
Good to go

**Pull request merged**